lib/tasks/manageiq/webpacker.rake

```diff
@@ -1,33 +1,34 @@
-if File.exist? 'manageiq-ui-classic.gemspec'
-  # all of these tasks are only internal ui-classic, used by webpack:server, webpack:compile or webpack:clobberr
-  # mostly because they fail when run from manageiq/
-
-  # yarn:install is a rails 5.1 task, webpacker:compile needs it
-  namespace :yarn do
-    #task :install => ["update:yarn"]
+# yarn:install is a rails 5.1 task, webpacker:compile needs it
+namespace :yarn do
+  task :install do
     puts 'yarn:install called, not doing anything'
   end
+end
 
-  # need the initializer for the rake tasks to work
-  require ManageIQ::UI::Classic::Engine.root.join('config/initializers/webpacker.rb')
+# need the initializer for the rake tasks to work
+require ManageIQ::UI::Classic::Engine.root.join('config/initializers/webpacker.rb')
+unless Rake::Task.task_defined?("webpacker")
   load 'tasks/webpacker.rake'
   load 'tasks/webpacker/compile.rake'
   load 'tasks/webpacker/clobber.rake'
-  load 'tasks/webpacker/verify_install.rake'  # needed by compile
-  load 'tasks/webpacker/check_node.rake'  # needed by verify_install
-  load 'tasks/webpacker/check_yarn.rake'  # needed by verify_install
-  load 'tasks/webpacker/check_webpack_binstubs.rake'  # needed by verify_install
+  load 'tasks/webpacker/verify_install.rake'         # needed by compile
+  load 'tasks/webpacker/check_node.rake'             # needed by verify_install
+  load 'tasks/webpacker/check_yarn.rake'             # needed by verify_install
+  load 'tasks/webpacker/check_webpack_binstubs.rake' # needed by verify_install
+end
 
-  namespace :webpack do
-    task :paths do
-      filename = ManageIQ::UI::Classic::Engine.root.join('config/webpack/paths.json')
-      File.open(filename, 'w') do |file|
-        file.write(JSON.pretty_generate({
-          :info    => "This file is autogenerated by rake webpack:paths, do not edit",
-          :output  => Rails.root.to_s,
-          :engines => asset_engines,
-        }) + "\n")
-      end
+namespace :webpack do
+  paths_config = ManageIQ::UI::Classic::Engine.root.join('config/webpack/paths.json')
+
+  file paths_config do |task|
```

Comment:
This means the file would never change again. Definitely can't have that, we *need* to rewrite the file if the list of engines has changed (or the paths).